sourcerer:0.2.1

miestrode opened the pull request:
<!--
Thanks for submitting a package! Please read and follow the submission guidelines detailed in the repository's README and check the boxes below. If you want to make a PR for something other than a package submission, just delete all this and make a plain PR.
-->

I am submitting
- [ ] a new package
- [x] an update for a package

<!--
Please add a brief description of your package below and explain why you think it is useful to others. If this is an update, please briefly say what changed.
-->

Description: The last version had a bug with it's lang-box, which was placed on the last page of the span of the code block. This is now fixed.

laurmaedje commented on packages/preview/sourcerer/0.2.1/README.md:
````diff
@@ -0,0 +1,35 @@
+# Usage
+First, import the package via:
+```typ
+#import "@preview/sourcerer:0.2.0": code
````
```suggestion
#import "@preview/sourcerer:0.2.1": code
```

miestrode commented:
Alright, I've fixed the `README.md` thing. Apparently the old release also had a bug with the alignment of the table parts, when you centered things for example, but this was also fixed by the new version here.
